.env

```diff
@@ -1,4 +1,4 @@
-DOCKER_REGISTRY_HOST=docker.io
+DOCKER_REGISTRY_HOST=ghcr.io
 DOCKER_IMAGE=nimblehq/google_scraper_ruby
```

Comment:
Should we change this to `aetheryna/google_scraper_ruby`?

Reply:
Your right! updated in https://github.com/aetheryna/google_scraper_ruby/pull/12/commits/91ef4f69dc5f179565fa15a9d2f7b1b64189cc8a 😄